app/scrapers/generalized.py

```diff
@@ -64,16 +68,22 @@ def search(self, query, num_results, qtype=''):
         """
         urls = []
         current_start = self.defaultStart
-
         while (len(urls) < num_results):
-            response = self.get_page(query, current_start, qtype)
+            response, status_code = self.get_page(query, current_start, qtype)
+            if response is None:
+                if(len(urls) == 0):
+                    return (None, status_code)
+                else:
+                    print("Couldn't fetch more results.")
```

Comment:
same

Reply:
This is the case where, urls fetched are not equal to expected number of urls. Should not be removed from logs then in my opinion. As we have not implemented `no. of urls fetched` yet.

Reply:
I am not asking to remove this, I am asking to replace with logging statements